hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/SparkBroadcastManager.java

```diff
@@ -116,10 +119,8 @@ public Option<HoodieReaderContext> retrieveFileGroupReaderContext(StoragePath ba
     SparkParquetReader sparkParquetReader = parquetReaderBroadcast.getValue();
     if (sparkParquetReader != null) {
       List<Filter> filters = new ArrayList<>();
-      return Option.of(new SparkFileFormatInternalRowReaderContext(
-          sparkParquetReader,
-          JavaConverters.asScalaBufferConverter(filters).asScala().toSeq(),
-          JavaConverters.asScalaBufferConverter(filters).asScala().toSeq()));
+      return Option.of(new SparkFileFormatInternalRowReaderContext(sparkParquetReader, JavaConverters.asScalaBufferConverter(filters).asScala().toSeq(),
```

Comment:
Maybe we follow the original indentation.

Reply:
Not part of this PR. Removed it.